### ISSUE TITLE
Failed to create the K8s client while listing Deprecated objects

### ISSUE THREAD

**Issue by mouhsen-ibrahim** [rikatz/kubepug#211]
Hi I just installed the latest version of `kubepug` 

```
GitVersion:    devel
GitCommit:     42529d7c573cbc2f5cb8d4810b715b29b76e0289
GitTreeState:  clean
BuildDate:     '2021-11-29T01:19:53Z'
GoVersion:     go1.17.3
Compiler:      gc
Platform:      linux/amd64
```

When I try to use it on my kubernetes cluster I get this error

```
> kubepug --k8s-version=v1.22.2
time="2021-11-29T09:13:06+01:00" level=fatal msg="Failed to create the K8s client while listing Deprecated objects"
```

It was working before I install the latest version, my cluster is operational and here is the version

```
> kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:28:09Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19+", GitVersion:"v1.19.14-gke.1900", GitCommit:"abc4e63ae76afef74b341d2dba1892471781604f", GitTreeState:"clean", BuildDate:"2021-09-07T09:21:04Z", GoVersion:"go1.15.15b5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by rikatz** [comment]
Oh did I added a recursion? 🤦‍♂️ let me check here

**Comment by rikatz** [comment]
@mouhsen-ibrahim I trieed here in my local KinD cluster and everything just works :(

Can you please paste somewhere the output of `./kubepug --verbosity=debug` ?

Thanks!

**Comment by rikatz** [comment]
While that, I will release a new kubepug version with better error outputing

**Comment by mouhsen-ibrahim** [comment]
@rikatz thanks for your response, I need first to have a look at the output and make sure it is safe to share outside my company after that I can post it here. Thanks again and have a nice day

**Comment by rikatz** [comment]
No worries @mouhsen-ibrahim 

Instead, wait until I release this new version with better outputing (doing this right now!) and LMK

**Comment by rikatz** [comment]
I just released v1.3.1 (go releaser is running), so as soon as this is released, if you can run it again at least we will have the error output of k8s client (why it cannot connect!) 

No need to send me the debug :)

**Comment by mouhsen-ibrahim** [comment]
@rikatz here is the new error output

```
time="2021-11-29T13:30:31+01:00" level=fatal msg="Failed to create the K8s client while listing Deprecated objects: no Auth Provider found for name \"gcp\""
```

**Comment by rikatz** [comment]
https://github.com/rikatz/kubepug/pull/213 :)

In some moment we removed the auth plugins, not sure when it happened... Readding should fix this!

Thanks for reporting!

**Comment by rikatz** [comment]
Fixed by #213 
/close
